src/boot/errors.r

```diff
@@ -219,6 +219,7 @@ User: [
 Internal: [
 	code: 900
 	type: "internal error"
+	misc:				[{miscellaneous error (add specific case to errors.r)}]
```

Comment:
Crash() had an RP_MISC you could use as a way of leaving a note about a crash that needed documenting, but errors didn't have a RE_MISC to draw attention to something that you didn't want to necessarily cause a crash for if it happened.  This lets you Trap with (RE_MISC)